doc/project/checklist.md

```diff
@@ -256,6 +256,11 @@ A testplan has been written (in HJson format) indicating:
 - The functional coverage plan captured as a list of covergroups, with a description highlighting which feature is expected to be covered by each covergroup.
   It may optionally contain additional details such as coverpoints and crosses of individual aspects of the said feature that is covered.
 
+If the DUT has a CPU for which a ROM firmware is developed (burnt-in during manufacturing):
+- A detailed ROM firmware testplan has been written to adequately verify all functions of the ROM firmware in a pre-silicon simulation environment (i.e. a DV testbench).
+- The testing framework may validate each functionality of the ROM firmware discreetly as an individual unit test.
```

Comment:
Nit: This should be "discretely" (at the moment, you're asking the testing framework to keep it on the down low :-p)

Reply:
lol, fixed :)